services/ui-src/src/components/reports/ModalOverlayReportPage.tsx

```diff
@@ -199,7 +199,7 @@ export const ModalOverlayReportPage = ({
         <Box sx={sx.content}>
           <ReportPageIntro
             text={verbiage.intro}
-            reportType={report?.reportType}
+            reporttype={report?.reportType}
```

Comment:
same question here about advantage of having this all lowercase in DOM

Reply:
It looks like the same thing is happening here where `reportType/reporttype` is never used as a property in ReportPageIntro so I would just remove it here

Reply:
Yess! I can remove `reportType` 🙂